handlers/k8sconfig_handler.go

```diff
@@ -325,22 +325,20 @@ func (h *Handler) setupK8sConfig(inClusterConfig bool, k8sConfigBytes []byte, co
 	kc.ClusterConfigured = true
 	return kc, nil
 }
-func registerK8sComponents(l logger.Handler, config []byte, ctx string) error {
-	l.Info("Starting to register k8s native components")
+func registerK8sComponents(l *Handler, config []byte, ctx string) error {
+	l.log.Info("Starting to register k8s native components")
 	man, err := core.GetK8Components(config, ctx)
 	if err != nil {
 		return err
 	}
 	if man == nil {
-		l.Error(errors.New("Could not get k8s components"))
+		l.log.Error(errors.New("Could not get k8s components"))
 		return err
 	}
 	for i, def := range man.Definitions {
 		var ord core.WorkloadCapability
 		ord.Metadata = make(map[string]string)
 		ord.Metadata["io.meshery.ctxid"] = ctx
-		ord.Metadata["adapter.meshery.io/name"] = "kubernetes"
```

Comment:
@Revolyssup why are we getting rid of these?

Reply:
@utkarsh-pro  this PR is older than the recent changes in master that were made when these fields were added, originally these fields weren't there. That is why they are missing here, adding those fields accordingly now